lib/wallet/src/Cardano/Wallet/Write/Tx/Balance.hs

```diff
@@ -85,6 +85,8 @@ import Cardano.Wallet.Address.Derivation.Shared
     ( SharedKey (..) )
 import Cardano.Wallet.Primitive.Types
     ( TokenBundleMaxSize (TokenBundleMaxSize) )
+import Cardano.Wallet.Primitive.Types.Coin
+    ( Coin )
```

Comment:
Could we let `Coin` refer to `Write.Tx.Coin` and import this as `W.Coin` if we need to?